include/Oscillator.h

```diff
@@ -79,6 +83,14 @@ class EXPORT Oscillator
 		delete m_subOsc;
 	}
 
+	static void waveTableInit();
+	static void destroyFFTPlans();
```

Comment:
Can you move this line after `createFFTPlans` for consistency?

Reply:
Done.

Reply:
Okay.